web/modules/custom/contact_form/contact_form.module

```diff
@@ -117,3 +127,44 @@ function contact_form_mail_alter(&$message) {
     $message['from'] = $from;
   }
 }
+
+/**
+ * Implements hook_mail().
+ *
+ * Send a notification to the contact form submitter.
+ */
+function contact_form_mail($key, &$message, $params) {
+  if ($key !== 'contact_form_sender') {
+    return;
+  }
+
+  /** @var \Drupal\message\MessageInterface $entity */
+  $entity = $params['message'];
+  $name = "{$entity->field_contact_first_name->value} {$entity->field_contact_last_name->value}";
+
+  $message['subject'] = t('Joinup: Contact form notification');
+
+  $message['body'][] = t('Dear @name', ['@name' => $name]);
+  $message['body'][] = t('Thank you for submitting your message to the Joinup Support.');
+
+  if ($entity->field_contact_copy->value) {
+    $message['body'][] = t('This is the message you submitted through the online form:');
+    $subject = check_markup($entity->field_contact_subject->value, $entity->field_contact_subject->format);
+    $subject = PlainTextOutput::renderFromHtml($subject);
+    $message['body'][] = t('Subject: %subject', ['%subject' => $subject]);
+    $message['body'][] = t('Content:');
+    $message['body'][] = '<blockquote>';
+    $message['body'][] = check_markup($entity->field_contact_message->value, $entity->field_contact_message->format);
+    $message['body'][] = '</blockquote>';
+    $message['body'][] = t('Thank you for your interest, we will revert back to you as soon as possible.');
+  }
+  else {
+    $message['body'][] = t('We will revert back to you as soon as possible.');
```

Comment:
The expression "revert back to you" is not correct English, this should be something like "We will reply as soon as possible". I will confirm this with the functional team who is responsible for delivering the user facing strings.